classes/woocommerce-seo.php

```diff
@@ -435,14 +435,17 @@ public function add_submenu_pages( $submenu_pages ) {
 	}
 
 	/**
-	 * Loads CSS.
+	 * Loads CSS in Woocommerce SEO settings page.
 	 *
 	 * @since 1.0
 	 */
 	public function config_page_styles() {
 		global $pagenow;
 
-		$is_wpseo_woocommerce_page = ( $pagenow === 'admin.php' && filter_input( INPUT_GET, 'page' ) === 'wpseo_woo' );
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Nonce is not required here.
+		$is_get_wpseo_woo = ( isset( $_GET['page'] ) && is_string( $_GET['page'] ) && $_GET['page'] === 'wpseo_woo' );
```

Comment:
And if we're strictly checking, there is no need for the string check 😄 
```suggestion
		$is_get_wpseo_woo = ( isset( $_GET['page'] ) && $_GET['page'] === 'wpseo_woo' );
```

Reply:
Right!